JavaAssignment/src/ankitaG/Program8.java

```diff
@@ -0,0 +1,27 @@
+package ankitaG;
+
+public class Program8 {
+	static int num1, num2, num3;
+
+	static void setdata(int x, int y)
```

Comment:
Again the same comment I gave to you in https://github.com/krishnakTechnocretids/JAVATechnoCreditsG4/pull/29 PR.